packages/form-layout/src/vaadin-form-layout.js

```diff
@@ -542,6 +538,11 @@ class FormLayout extends ElementMixin(ThemableMixin(PolymerElement)) {
         }
       });
   }
+
+  /** @protected */
```

Comment:
Let's add `@override` also, to indicate that we override a method inherited from the mixin.

Reply:
Done